src/mscorlib/src/System/Runtime/Intrinsics/X86/Avx2.cs

```diff
@@ -352,6 +352,11 @@ public static class Avx2
         /// </summary>
         public static Vector256<ulong> ConvertToVector256ULong(Vector128<uint> value) => ConvertToVector256ULong(value);
```

Comment:
note: This is an example of an API that doesn't follow the general .NET API naming conventions. It should probably be `ConvertToVector256Int64`